src/ports/postgres/modules/sample/balance_sample.sql_in

```diff
@@ -149,8 +149,10 @@ non-stratified, that is, the whole table is treated as a
 single group.
 
 @note
-Current implementation does not support grouping_cols.  It
-will be added in an upcoming release.
+The 'output_table_size' and the 'class_sizes' are defined for the whole table.
+When grouping is used, these parameters are split evenly for each group.
+Further, if a specific class value is specified in the 'class_sizes' parameter,
+that particular class value should be present in each group.
 </dd>
```

Comment:
Please change/add:
"...that particular class value should be present in each group."
to 
"...that particular class value should be present in each group.  If not, an error will be thrown."